cmake/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@
 # Licensed under the MIT License.
 
 # Minimum CMake required
-cmake_minimum_required(VERSION 3.11)
+cmake_minimum_required(VERSION 3.8)
```

Comment:
Wait, let me verify it.
@KeDengMS Why did you set this to 3.11?

Reply:
Hi @apwojcik, our cuda build requires cmake 3.11+,  could you please left this line untouched?